lib/pf/UnifiedApi.pm

```diff
@@ -69,7 +69,14 @@ our @API_V1_ROUTES = (
     { controller => 'ApiUsers' },
     { controller => 'Locationlogs' },
     { controller => 'Config::ConnectionProfiles' },
-    { controller => 'Violations', },
+    { 
+        controller => 'Violations',
+        collection => {
+            subroutes    => {
+                ':search' => { get => 'search' },
```

Comment:
I think based off the discussions with @jrouzierinverse this should be named differently